packages/amplication-client/src/VersionControl/BuildPage.scss

```diff
@@ -24,9 +24,10 @@ $icon-size: 20px;
   }
   &__build-details {
     display: flex;
-    direction: row;
+    flex-direction: row;
 
     .log-container {
+      height: 500px;
```

Comment:
The height should not be fixed and should be used relatively to the page content. we can consider a parent with 100% and then flex 1 to use the entire space left by the siblings 

let me know if you need more details on instructions on this one